hub/core/storage/lru_cache.py

```diff
@@ -182,7 +190,7 @@ def clear_cache(self):
         self.dirty_keys.clear()
         self.cache_storage.clear()
 
-        if hasattr(self.next_storage, "clear_cache"):
+        if self.next_storage is not None and hasattr(self.next_storage, "clear_cache"):
```

Comment:
when will it not have a clear_cache method? shouldn't we just have an empty method in those cases anyways since it's a subclass?

Reply:
next_storage could be a LocalProvider or a MemoryProvider which isn't a cache, so makes no sense to add clear_cache as a method for them